tests/README.md

```diff
@@ -223,6 +223,38 @@ EXTERNALSRC_pn-mender-artifact-native = "$GOPATH"
 > Remember to add your '$GOPATH' in the conf file, it is not taken from the environment.
 
 
+## Run the integration tests in the staging environment running in Kubernetes
+
+It is possible to run the integration tests targeting the staging environment running in Kubernetes.
+You can either run them using GitLab (recommended) or from your local environment.
+
+### Run the integration tests using GitLab
+
+To run the tests in GitLab, [start a new pipeline in GitLab](https://gitlab.com/Northern.tech/Mender/integration/-/pipelines/new), select your branch (`staging`, for example) and set the `RUN_TESTS_STAGING` variable to `true`.
+
+### Run the tests from your local environment
+
+In order to do it, you need to export the following environment variables:
+
+```bash
+$ export K8S="staging"
+$ export AWS_ACCESS_KEY_ID="<aws-access-key>"
+$ export AWS_SECRET_ACCESS_KEY="<aws-access-key>"
+$ export AWS_DEFAULT_REGION="us-east-1"
+$ export AWS_EKS_CLUSTER_NAME="hosted-mender-staging"
+$ export GATEWAY_HOSTNAME="staging.hosted.mender.io"
+```
+
+The values of the variables follow:
+
+* **K8S** contains the name of the namespace where the Mender product is running in the Kubernetes cluster;
+* **AWS_ACCESS_KEY_ID** and **AWS_SECRET_ACCESS_KEY** are the AWS access key and secret, used to authenticate to the EKS cluster;
+* **AWS_DEFAULT_REGION** contains the region where the EKS cluster is running;
+* **AWS_EKS_CLUSTER_NAME** contains the name of the EKS cluster;
+* **GATEWAY_HOSTNAME** determines the Mender API gateway's public host name, accessible via HTTPS, to call public API end-points.
+
+You can now start the test using the `run.sh script.
```

Comment:
To avoid confusion (like happened to me), please add a note like:

Note that you can only run Enterprise tests against staging, so either use `./run.sh -- -k Enteprise` or set environment variable `SPECIFIC_INTEGRATION_TEST: "Enterprise"`